gui/api/resources.py

```diff
@@ -920,12 +922,24 @@ def _get_children(self, bundle, vol, children, uid):
                     data['used_pct'],
                 )
                 data['avail'] = humanize_size(data['avail'])
-                
+
                 data['readonly'] = self.__zfsopts.get(
                     child.path,
                     {},
                 ).get('readonly', ('', '-'))[1]
 
+                data['comments'] = self.__zfsopts.get(
+                    child.path,
+                    {},
+                ).get('org.freenas:description', ('', '-'))[1]
+
+                # Filtering out comments field for Jails
```

Comment:
Why are you filtering by jail? That doesn't make sense. Its a feature self-contained.